component/src/main/java/io/siddhi/extension/store/elasticsearch/ElasticsearchConfigs.java

```diff
@@ -0,0 +1,532 @@
+        bulkProcessorBuilder.setBulkSize(new ByteSizeValue(bulkSize, ByteSizeUnit.MB));
+        bulkProcessorBuilder.setConcurrentRequests(concurrentRequests);
+        bulkProcessorBuilder.setFlushInterval(TimeValue.timeValueSeconds(flushInterval));
+        if (backoffPolicy == null) {
+            bulkProcessorBuilder.setBackoffPolicy(BackoffPolicy.noBackoff());
+        } else if (backoffPolicy.equalsIgnoreCase(ANNOTATION_ELEMENT_BACKOFF_POLICY_CONSTANT_BACKOFF)) {
+            bulkProcessorBuilder.setBackoffPolicy(BackoffPolicy.constantBackoff(
+                    TimeValue.timeValueSeconds(backoffPolicyWaitTime), backoffPolicyRetryNo));
+        } else {
+            bulkProcessorBuilder.setBackoffPolicy(BackoffPolicy.exponentialBackoff(
+                    TimeValue.timeValueSeconds(backoffPolicyWaitTime), backoffPolicyRetryNo));
+        }
+        bulkProcessor = bulkProcessorBuilder.build();
+    }
+
+    public RestHighLevelClient getRestHighLevelClient() {
+        return restHighLevelClient;
+    }
+
+    public void setRestHighLevelClient(RestHighLevelClient restHighLevelClient) {
+        this.restHighLevelClient = restHighLevelClient;
+    }
+
+    public List<Attribute> getAttributes() {
+        return attributes;
+    }
+
+    public String getHostname() {
+        return hostname;
+    }
+
+    public String getIndexName() {
+        return indexName;
+    }
+
+    public void setIndexName(String indexName) {
+        this.indexName = indexName;
+    }
+
+    public String getIndexAlias() {
+        return indexAlias;
+    }
+
+    public int getPort() {
+        return port;
+    }
+
+    public int getNumberOfShards() {
+        return numberOfShards;
+    }
+
+    public int getNumberOfReplicas() {
+        return numberOfReplicas;
+    }
+
+    public BulkProcessor getBulkProcessor() {
+        return bulkProcessor;
+    }
+
+    public int getPayloadIndexOfIndexName() {
+        return payloadIndexOfIndexName;
+    }
+
+    public String getDefinitionId() {
+        return definitionId;
+    }
+
+    public Annotation getStoreAnnotation() {
+        return storeAnnotation;
+    }
+
+    static class BulkProcessorListener implements BulkProcessor.Listener {
+
+        private ElasticsearchSink elasticsearchSink;
+
+        private BulkProcessorListener(ElasticsearchSink elasticsearchSink) {
+            this.elasticsearchSink = elasticsearchSink;
+        }
+
+        private BulkProcessorListener() {
+
+        }
+
+        @Override
+        public void beforeBulk(long executionId, BulkRequest request) {
+
+            int numberOfActions = request.numberOfActions();
+            if (logger.isDebugEnabled()) {
+                logger.debug("Executing bulk [{" + executionId + "}] with {" + numberOfActions + "} requests");
+            }
+        }
+
+        @Override
+        public void afterBulk(long executionId, BulkRequest request, BulkResponse response) {
+
```

Comment:
Don't have spaces between the method content and method definition.

Reply:
Fixed in https://github.com/siddhi-io/siddhi-store-elasticsearch/pull/41/commits/2c90b5856b203a9ad479c745f00b96442646fa04